### PR TITLE
Update repo to v5

### PR DESCRIPTION
**Pull request by Araknyfe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,20 +9,23 @@ Types in `null` will only be considered null on null input, and will JSON encode
...
 
-### null package
+- All types implement `sql.Scanner` and `driver.Valuer`, so you can use this library in place of `sql.NullXXX`.
+- All types also implement `json.Marshaler` and `json.Unmarshaler`, so you can marshal them to their native JSON representation.
+- All non-generic types implement `encoding.TextMarshaler`, `encoding.TextUnmarshaler`. A null object's `MarshalText` will return a blank string.
 
-`import "gopkg.in/guregu/null.v4"`
+## null package
+
+`import "github.com/guregu/null/v5"`
```
Comment (System-Glitch) [inline README.md]:
```suggestion
`import "github.com/onfocusio/null"`
```
Can you fix the other imports too please? I just realized this doc is incorrect for our fork.

File: README.md [hunk 2]
```diff
@@ -74,17 +82,35 @@ Nullable bool.
...
-*As of v4*, unmarshaling from JSON `sql.NullXXX` JSON objects (ex. `{"Int64": 123, "Valid": true}`) is no longer supported. It's unlikely many people used this, but if you need it, use v3.
+
+#### v5
+- Now a Go module under the path `github.com/guregu/null/v5`
+- Added missing types from `database/sql`: `Int32, Int16, Byte`
+- Added generic `Value[T]` embedding `sql.Null[T]`
+
+#### v4
+- Available at `gopkg.in/guregu/null.v4`
+- Unmarshaling from JSON `sql.NullXXX` JSON objects (e.g. `{"Int64": 123, "Valid": true}`) is no longer supported. It's unlikely many people used this, but if you need it, use v3.
```
Comment (System-Glitch) [inline README.md]: This section can be removed (or updated to explain this is a simple fork of `github.com/guregu/null` that contains additional types.

File: float.go [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
-	"bytes"
 	"database/sql"
 	"encoding/json"
-	"errors"
 	"fmt"
 	"math"
 	"reflect"
 	"strconv"
+
+	"github.com/guregu/null/internal"
```
Comment (System-Glitch) [inline float.go]:
```suggestion
	"github.com/onfocusio/null/internal"
```

File: int.go [hunk 4]
```diff
@@ -1,12 +1,10 @@
...
 
 import (
-	"bytes"
 	"database/sql"
-	"encoding/json"
-	"errors"
-	"fmt"
 	"strconv"
+
+	"github.com/guregu/null/internal"
```
Comment (System-Glitch) [inline int.go]:
```suggestion
	"github.com/onfocusio/null/internal"
```

File: int16.go [hunk 5]
```diff
@@ -1,22 +1,20 @@
...
 
 import (
-	"bytes"
 	"database/sql"
-	"encoding/json"
-	"errors"
-	"fmt"
 	"strconv"
+
+	"github.com/guregu/null/v5/internal"
```
Comment (System-Glitch) [inline int16.go]:
```suggestion
	"github.com/onfocusio/null/internal"
```

File: int32.go [hunk 6]
```diff
@@ -0,0 +1,107 @@
+package null
+
+import (
+	"database/sql"
+	"strconv"
+
+	"github.com/guregu/null/internal"
```
Comment (System-Glitch) [inline int32.go]:
```suggestion
	"github.com/onfocusio/null/internal"
```

File: int_test.go [hunk 7]
```diff
@@ -1,269 +1,393 @@
...
 
 import (
+	"encoding"
 	"encoding/json"
 	"errors"
 	"math"
 	"strconv"
 	"testing"
+
+	"github.com/guregu/null/internal"
```
Comment (System-Glitch) [inline int_test.go]:
```suggestion
	"github.com/onfocusio/null/internal"
```

File: value_test.go [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+
+import (
+	"bytes"
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/guregu/null/internal"
```
Comment (System-Glitch) [inline value_test.go]:
```suggestion
	"github.com/onfocusio/null/internal"
```

File: zero/byte.go [hunk 9]
```diff
@@ -0,0 +1,119 @@
+package zero
+
+import (
+	"database/sql"
+	"strconv"
+
+	"github.com/guregu/null/internal"
```
Comment (System-Glitch) [inline zero/byte.go]:
```suggestion
	"github.com/onfocusio/null/internal"
```

File: zero/float.go [hunk 10]
```diff
@@ -1,14 +1,14 @@
...
-	"bytes"
 	"database/sql"
 	"encoding/json"
-	"errors"
 	"fmt"
 	"math"
 	"reflect"
 	"strconv"
+
+	"github.com/guregu/null/internal"
```
Comment (System-Glitch) [inline zero/float.go]:
```suggestion
	"github.com/onfocusio/null/internal"
```

File: zero/int.go [hunk 11]
```diff
@@ -1,12 +1,10 @@
...
 
 import (
-	"bytes"
 	"database/sql"
-	"encoding/json"
-	"errors"
-	"fmt"
 	"strconv"
+
+	"github.com/guregu/null/internal"
```
Comment (System-Glitch) [inline zero/int.go]:
```suggestion
	"github.com/onfocusio/null/internal"
```

File: zero/int16.go [hunk 12]
```diff
@@ -0,0 +1,119 @@
+package zero
+
+import (
+	"database/sql"
+	"strconv"
+
+	"github.com/guregu/null/internal"
```
Comment (System-Glitch) [inline zero/int16.go]:
```suggestion
	"github.com/onfocusio/null/internal"
```

File: zero/int32.go [hunk 13]
```diff
@@ -0,0 +1,119 @@
+package zero
+
+import (
+	"database/sql"
+	"strconv"
+
+	"github.com/guregu/null/internal"
```
Comment (System-Glitch) [inline zero/int32.go]:
```suggestion
	"github.com/onfocusio/null/internal"
```

File: zero/int_test.go [hunk 14]
```diff
@@ -1,87 +1,117 @@
...
 
 import (
+	"encoding"
 	"encoding/json"
 	"errors"
 	"math"
 	"strconv"
 	"testing"
+
+	"github.com/guregu/null/internal"
```
Comment (System-Glitch) [inline zero/int_test.go]:
```suggestion
	"github.com/onfocusio/null/internal"
```

File: zero/value_test.go [hunk 15]
```diff
@@ -0,0 +1,197 @@
...
+
+import (
+	"bytes"
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/guregu/null/internal"
```
Comment (System-Glitch) [inline zero/value_test.go]:
```suggestion
	"github.com/onfocusio/null/internal"
```

**Review by System-Glitch — CHANGES_REQUESTED** [review]
Need to replace the condition in `interval.go:40` to
```diff
func (i *Interval) UnmarshalJSON(data []byte) error {
-	if bytes.Equal(data, nullBytes) {
+	if len(data) > 0 && data[0] == 'n' { 
		i.Status = pgtype.Null
		return nil
	}
```

File: README.md (hunk 1 above)
Comment (Araknyfe) [reply README.md]: Should all be good :D

File: int16.go (hunk 5 above)
Comment (System-Glitch) [reply int16.go]: You forgot this one. Otherwise LGTM

**Review by System-Glitch — APPROVED** [review]

**Pull request merged**